e2e/utils/waits-utils.ts

```diff
@@ -177,7 +178,7 @@ export async function waitForAttributeEquality(page: Page,
           return element.attributes[attributeName] && element.attributes[attributeName].value === attributeValue;
         }
         return false;
-      }, {}, selector.css, attribute, value);
+      }, timeout ? {timeout} : {}, selector.css, attribute, value);
```

Comment:
line 181 can be rewritten to `}, {timeout}, selector.css, attribute, value);`

Reply:
eek actually I think it should be `{timeout: timeout || 30000}`